ow_lander/scripts/activity_guarded_move.py

```diff
@@ -37,8 +34,8 @@ def arg_parsing(req):
   return [req.use_defaults, target_x, target_y, target_z,
           direction_x, direction_y, direction_z, search_distance]
 
-# Approach
-def pre_guarded_move(move_arm, args): 
+
+def pre_guarded_move(move_arm, args):
   """
```

Comment:
"Approach" wasn't very descriptive, but it would be good to have comments that explain the distinction between pre_guarded_move and guarded_move_plan.

Reply:
`# Approach` was somewhat confusing to me. Did it mean the method to solve a problem or it  actually meant approaching a given point. As you said it is not descriptive at all. I'll try to elaborate a bit in the function summary.